.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 20.9.0
```

Comment:
Added a `.tool-versions` file for folks who use other version managers like `asdf`